front/src/modules/pathfinding/helpers/findTrackSectionOffset.ts

```diff
@@ -0,0 +1,36 @@
+import type { TrackRange } from 'common/api/osrdEditoastApi';
+
+const findTrackSectionIndex = (position: number, tracksLengthCumulativeSums: number[]) => {
+  for (let i = 0; i < tracksLengthCumulativeSums.length; i += 1) {
+    if (position <= tracksLengthCumulativeSums[i]) {
+      return i;
+    }
+  }
+  return -1;
+};
+
+/**
+ * Return the track offset corresponding to the given position on path, composed of the track section id
+ * and the offset on this track section in mm
+ */
+const findTrackSectionOffset = (
+  positionOnPath: number, // in mm
+  tracksLengthCumulativeSums: number[], // in mm
+  trackRangesOnPath: TrackRange[]
+) => {
+  const index = findTrackSectionIndex(positionOnPath, tracksLengthCumulativeSums);
```

Comment:
Wouldn't it work to just do that ?
```suggestion
  const index = tracksLengthCumulativeSums.findIndex(
    (cumulativeSum) => positionOnPath <= cumulativeSum
  );
```

Reply:
Yes you're right, it's better like this :)